### ISSUE TITLE
test 

### ISSUE THREAD

**Issue by dfx1822375** [timeseriesAI/tsai#485]
Hi @oguiza:
I would like to know some questions about testing，and how do I test the test set after training the model? 
My training accuracy is about 80, why the test set accuracy is very low？
Code is here：
‘’‘dsets = TSDatasets(X_feat_new, Y_training, tfms=tfms, splits=splits)
dls = TSDataLoaders.from_dsets(dsets.train, dsets.valid, bs=bs, batch_tfms=TSStandardize(by_var=True))
model = TST(dls.vars, dls.c, dls.len,dropout=0.3,fc_dropout=0.9)
learn = Learner(dls, model, loss_func=LabelSmoothingCrossEntropyFlat(), metrics=[RocAuc(),accuracy])
learn.fit_one_cycle(35, 0.0015964782268274575) # 0.0001
probas, _, preds = learn.get_X_preds(X_test_feat_new)
print('test_label',test_label)
print('predictions',preds)’''
because I don't know fastai.data.transforms.Category how to convert to Numpy, so I print them and watch them.
The above code is what I got from "10_Time_Series_Classification_and_Regression_with_MiniRocket.ipynb"  read,but I don't  know what is 'skm.accuracy_score',the code in the original article is as follows:
'''probas, _, preds = learn.get_X_preds(new_feat)
skm.accuracy_score(y[splits[1]], preds)’‘’

**Comment by oguiza** [comment]
Sorry, I'm unable to understand the code you've posted.
Please, replace skm by sklearn.metrics.
Any code you paste needs to be between 3 backticks ``` code```. You can see how others will see your message just by clicking Preview when you are writing the code.

**Comment by dfx1822375** [comment]
> Sorry, I'm unable to understand the code you've posted. Please, replace skm by sklearn.metrics. Any code you paste needs to be between 3 backticks ` code`. You can see how others will see your message just by clicking Preview when you are writing the code.

Hi @oguiza :
    I am very sorry for the trouble I caused you because of my stupidity.
    I just want to know how to test the trained model on the test set in tsai, and in my code I have about 80 percent accuracy on the training set, but my test set accuracy is lower, and I want to know why. And the type fastai.data.transforms.Category how to convert to Numpy.
I'm very sorry, I'm really not very good at markdown, the code is as follows：
`tfms  = [None, [Categorize()]]`
`splits= get_splits(Y_training, valid_size=.2, stratify=True,  show_plot=False,random_state=123)`
`dsets = TSDatasets(X_training, Y_training, tfms=tfms, splits=splits)`
`dls = TSDataLoaders.from_dsets(dsets.train, dsets.valid, bs=bs, batch_tfms=TSStandardize(by_var=True))`
`model = TST(dls.vars, dls.c, dls.len,dropout=0.3,fc_dropout=0.9)`
`learn = Learner(dls, model, loss_func=LabelSmoothingCrossEntropyFlat(), metrics=[RocAuc(),accuracy])`
`learn.fit_one_cycle(35, 0.0015964782268274575) `
`probas, _, preds = learn.get_X_preds(X_test_feat_new)`
`print('test_label',test_label)`
`print(predictions',preds)`

**Comment by oguiza** [comment]
There're no stupid questions @dfx1822375 .
In almost all cases training accuracy is higher than your validation accuracy. The important thing is to see if your validation accuracy was high during training, and then started to drop. In that case, that is called overfitting, and you should use regularization techniques, like data augmentation, weight decay (wd), dropout, etc. 
When you get the probas and press they are tensors. If you want to convert them to numpy arrays just do: probas.cpu().numpy().

**Comment by oguiza** [comment]
I'll close this issue due to a lack of response.
